README.md

```diff
@@ -1,5 +1,15 @@
 # Foreman OpenNebula Plugin
 
+**This plugin has been discontinued. The latest release only helps with its removal. If you're interested in taking over the maintanance, let us know.**
```

Comment:
Might be good to post to discourse and link it here so users know where to start.